packages/patternlab/styleguide/source/assets/js/modules/skipLinkTarget.js

```diff
@@ -33,25 +32,26 @@ document.querySelectorAll(".ma__figure__skip-link").forEach(link => {
       document.querySelector(location.hash).style.display = "none";
     }
   });
-});
```

Comment:
This `document.querySelectorAll(".ma__figure__skip-link").forEach(link => {` is for skip links for figures (currently, only set up for Tableau data viz. inserts) and unrelated to the code for the TOC skip links shown below .  So, it closed here.

Reply:
I see... I didn't realize that. good catch! I